src/lib/stores/wallet/wallet.store.ts

```diff
@@ -15,25 +15,77 @@ import globalAdvisoryStore from '../global-advisory/global-advisory.store';
 import SafeAppsSDK from '$lib/stores/wallet/safe/sdk';
 import { SafeAppProvider } from '@safe-global/safe-apps-provider';
 import isRunningInSafe from '$lib/utils/is-running-in-safe';
+import storedWritable from '@efstajas/svelte-stored-writable';
+import { z } from 'zod';
+import { isWalletUnlocked } from './utils/is-wallet-unlocked';
+import themeStore from '../theme/theme.store';
 
 const appsSdk = new SafeAppsSDK();
 
+const MAINNET_RPC_URL = 'https://mainnet.infura.io/v3/f88a1229d473471bbf94d168401b9c93';
+const GOERLI_RPC_URL = 'https://goerli.infura.io/v3/f88a1229d473471bbf94d168401b9c93';
```

Comment:
Don't we need `sepolia`?

Reply:
On main, we don't have Sepolia support yet, and this PR is going to main. I suggest once this is merged, we merge it back into `feature/dependency-funding` and while resolving the inevitable conflict add the Sepolia config.

Reply:
Ah right, always forgetting this... ☺️